crates/astria-conductor/src/executor/builder.rs

```diff
@@ -24,6 +24,7 @@ pub(crate) struct NoShutdown;
 pub(crate) struct WithShutdown(oneshot::Receiver<()>);
 
 pub(crate) struct ExecutorBuilder<TRollupAddress = NoRollupAddress, TShutdown = NoShutdown> {
+    consider_commitment_spread: bool,
```

Comment:
What's the purpose of providing the option in builder to not consider the commitment spread?

Reply:
You can only consider the spread if you are both syncing soft && firm - although you only have to really disable it for soft syncing as here the firm block num will always stay at 0.

Executor does not a priori know that it will never receive firm blocks in the soft-only case and so you have to disable the spread.